### PR TITLE
Submit cran

### PR DESCRIPTION
**Pull request by Tina815**

### CODE REVIEW

File: .Rbuildignore [hunk 1]
```diff
@@ -1,9 +1,11 @@
 ^phsmethods\.Rproj$
 ^\.Rproj\.user$
 ^README\.Rmd$
+^README\.html$
```
Comment (Moohan) [inline .Rbuildignore:4]: This shouldn't be needed now.
```suggestion
```

**Review by Moohan — APPROVED** [review]
All the changes  good to me 👍

File: .Rbuildignore (hunk 1 above)
Comment (Tina815) [reply .Rbuildignore:4]: Can we leave it there just in case?

Comment (Moohan) [reply .Rbuildignore:4]: Yep, fine to leave in, it would just be redundant but won't negatively affect anything.

**Pull request merged**